gradle/build_test.go

```diff
@@ -45,17 +47,31 @@ func testBuild(t *testing.T, context spec.G, it spec.S) {
 		ctx.Application.Path, err = ioutil.TempDir("", "build-application")
 		Expect(err).NotTo(HaveOccurred())
 
+		ctx.Buildpack.Metadata = map[string]interface{}{
+			"configurations": []map[string]interface{}{
+				{"name": "BP_GRADLE_BUILD_ARGUMENTS", "default": "test-argument"},
+			},
+		}
+
 		ctx.Layers.Path, err = ioutil.TempDir("", "build-layers")
 		Expect(err).NotTo(HaveOccurred())
 
+		homeDir, err = ioutil.TempDir("", "home-dir")
+		Expect(err).NotTo(HaveOccurred())
+
 		gradleBuild = gradle.Build{
 			ApplicationFactory: &FakeApplicationFactory{},
+			HomeDirectoryResolver: FakeHomeDirectoryResolver{path: homeDir},
 		}
 	})
 
 	it.After(func() {
 		Expect(os.RemoveAll(ctx.Application.Path)).To(Succeed())
 		Expect(os.RemoveAll(ctx.Layers.Path)).To(Succeed())
+
+		homeDir,err := gradleBuild.HomeDirectoryResolver.Location()
+		Expect(err).ToNot(HaveOccurred())
```

Comment:
What is the purpose of doing this in `it.After`?

Reply:
No purpose. I forgot to remove that. :) I think I had it in there from before I was storing the home directory. Removed it.